test/selenium/main.js

```diff
@@ -0,0 +1,82 @@
+// Before we do ANYTHING else, verify that there are drivers in the main dir.
+// If there aren't, then this is going to crash and burn, and whoever's trying
+// to run this should "install" the firefox and chrome drivers for their OS.
+
+const fs = require('fs');
+
+const geckoDriver = fs.existsSync('./geckoDriver') || fs.existsSync('./geckoDriver.exe');
```

Comment:
Could we use the env value from the .env and .docker-env instead? Just in case people want to put it elsewhere.

Reply:
Based on how the selenium webdriver code works. It _only_ works with the drivers either in the dir you run selenium, which is easy for people to effect, or with the webdrivers installed somewhere else and then with the system PATH updated, which should never be a requirement for individual projects.

Reply:
OK! So why do we need those two env vars for?

Reply:
telling selenium where to find the actual Firefox and Chrome executables, as the webdriver will need to fire those up in headless mode.